v3/bastardkb/dilemma_v2/dilemma_v2.json

```diff
@@ -0,0 +1,287 @@
+{
+  "name": "Dilemma v2",
+  "vendorId": "0xA8F8",
+  "productId": "0x1836",
+  "menus": [
+    "qmk_rgb_matrix"
+  ],

```

Comment:
`"keycodes": [ "qmk_lighting" ],`
you'll probably want to add this in

Reply:
Thanks for the suggestion.

Is using QMK _Lightning_ keycodes relevant to keyboards using RGB _Matrix_?

Either way, the other Bastard Keyboards definitions don't have this optional setting enabled, so I'd prefer to keep it consistent for now, and we can reevaluate in the future and bring this feature to all keyboards at once if necessary.

Reply:
I mean if you don't want your users to be able to use on-board controls at all then you can keep it off. I wont know why you wouldn't enable it because it will always be enabled whether or not you have an RGB matrix or RGB light subsystem.

Reply:
Thanks for the explanation. I've included your suggestion.